spec/features/users_can_view_project_information_spec.rb

```diff
@@ -14,16 +14,25 @@
     visit project_information_path(project_id)
 
     expect(page).to have_content("Project details")
-
-    delivery_officer_label = page.find("dt", text: "Delivery officer")
-    delivery_officer_label.ancestor(".govuk-summary-list__row").find("dd", text: "user@education.gov.uk")
+    page_has_project_information_list_row(label: "Delivery officer", information: "user@education.gov.uk")
 
     expect(page).to have_content("School details")
+    page_has_project_information_list_row(label: "Original school name", information: "Caludon Castle School")
+    page_has_project_information_list_row(label: "Old Unique Reference Number", information: "12345")
+
+    expect(page).to have_content("Local authority details")
+    page_has_project_information_list_row(label: "Local authority", information: "West Placefield Council")
+  end
+
+  private def page_has_project_information_list_row(label:, information:)
+    project_information_list = page.find(".govuk-grid-column-two-thirds")
```

Comment:
Using `.govuk-grid-column-two-thirds` might be a bit tenuous, but I didn't want to add any classes for the purpose of testing

Reply:
We could slap an ID on it instead of a class? Arguably more useful for things like using a URI fragment to skip to the content.